huxley/www/static/js/huxley/components/AdvisorAssignmentsView.js

```diff
@@ -56,11 +60,33 @@ var AdvisorAssignmentsView = React.createClass({
     }.bind(this));
     CountryStore.getCountries(function(countries) {
       var new_countries = {};
-      for (var i = 0; i <countries.length; i++) {
+      for (var i = 0; i < countries.length; i++) {
         new_countries[countries[i].id] = countries[i];
       }
       this.setState({countries: new_countries})
     }.bind(this));
+    DelegateStore.getDelegates(user.school.id, function(delegates) {
+      // assigned is to manage the relation between an assignment and its delegates.
+      var assigned = {};
```

Comment:
Might be worth extracting this into a function with an explanatory docblock. At first glance, I have no idea what this is attempting to do.